client/starwhale/api/_impl/dataset/model.py

```diff
@@ -998,3 +998,51 @@ def dataset(
         )
 
         return ds
+
+    @classmethod
+    def from_dict(
+        cls, name: str = "", data: dict = {}, field_selector: str = ""
```

Comment:
```
name: (str, required) The dataset name you would like to use.
```
so we make name argument required?  `name: str=""` -> `name: str` ?

Reply:
`data: dict = {}` --> `data: dict`  or `data: t.Optional[t.Dict] = None`?

Reply:
done